feature/experimental/qos/qos_egress_scheduler/otg_tests/egress_strict_priority_scheduler_test/README.md

```diff
@@ -0,0 +1,93 @@
+# QoS-1.3: Egress strict priority scheduler test
+
+## Summary
+
+Verify that packet drops in AF4, AF3, AF2, AF1, BE1 and NC1 according to the strict-priority test traffic table
+
+## QoS traffic test setup:
+
+*   Topology:
+
+    *   2 input interfaces and 1 output interface with the same port speed. The
+        interface can be a physical interface or LACP bundle interface with the
+        same aggregated speed.
+
+    ```
+      ATE port 1
+          |
+         DUT--------ATE port 3
+          |
+      ATE port 2
+    ```
+
+*   Counters should be also verified for each test case:
+
+    *   /qos/interfaces/interface/output/queues/queue/state/transmit-pkts
+    *   /qos/interfaces/interface/output/queues/queue/state/dropped-pkts
+    *   transmit-pkts should be equal to the number of Rx pkts on Ixia port
+    *   dropped-pkts should be equal to diff between the number of Tx and the
+        number Rx pkts on Ixia ports
+
+*   Latency:
+
+    *   Should be < 100000ns
+
+## Procedure
```

Comment:
Why is this readme different from the one shared in https://docs.google.com/document/d/1zklh4PBVCGlhmGGAlb3beiy7KS9uypnZOOp4ZYiZpb8/edit?

Reply:
Addressed comment